src/networking.c

```diff
@@ -1147,6 +1156,14 @@ void addReplyBulkLongLong(client *c, long long ll) {
     addReplyBulkCBuffer(c, buf, len);
 }
 
+void addReplyBulkLongLongSkipPrepareClient(client *c, long long ll) {
```

Comment:
So, I thought about this a bit, and I have a suggestion. 

All of the `addReply*` functions call prepare client to write, so you don't have to worry about if the client is prepared to write. Maybe we can introduce a function like `writeReadyClient initClientReply(client *c);`, where the `writeClient` is just a special type where we know the client is prepared. Then this function becomes:

```
void addReplyBulkLongLong(writeReadyClient *c, long long ll) {
    ... blah blah ...
}
```

I would also like some type of special prefix like, `addWriteReadyReply` just they all have the same prefix. What this does is provide some type checking, so that nobody accidentally forgets to init.